.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,46 @@
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+| Name  | about |title|labels|
+| ------------- | ------------- |------------- |------------- |
+| Bug Names  | Create a report to help us improve stellar-core |[Short Description] (Version: [stellar-core version])|bug|
+## Read before creating an issue
+In general, we close any issues that are
```

Comment:
Can you revise this part as shown below?

In general, we close any issues that are

- unactionable (fill the template below under "Description")
- questions best served elsewhere

Reply:
I cannot see any changes in this part. Can you revise it again?